### PR TITLE
Create lock@8.0.4

### PR DESCRIPTION
**Pull request by Zoltu**
Let me know if there is a better way to have an override apply to multiple versions.

### CODE REVIEW

**Comment by Zoltu** [comment]
Getting it from bower appears to work out of the box, so this is unnecessary.  `jspm install bower:auth0-lock`

**Comment by guybedford** [comment]
The `bower` registry in jspm is really not encouraged. Happy to merge this.

**Comment by Zoltu** [comment]
Fixed broken build and re-opened PR.

File: package-overrides/github/auth0/lock@8.0.4.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "format": "global",
+  "main": "build/auth0-lock.min",
```
Comment (guybedford) [inline package-overrides/github/auth0/lock@8.0.4.json]: Usually its best to use the non-minified version for debugging.

**Comment by guybedford** [comment]
Did you try `npm:auth0-lock` here? Does that pull in too much cruft? It may be possible to remove the other stuff with an override similarly.

**Comment by Zoltu** [comment]
`npm:auth0-lock` doesn't appear have the pre-built libraries (no build folder or similar).  I have switched to un-minified source.

**Pull request merged**